Add better description to the tests (#31)
palaimon/ipyannotator: README.md

### Run ipyannotator tests

> Ipyannotator's tests were written on the Jupyter Notebook, following the literate programming style used on the library, and can be executed by running `nbdev_test_nbs` on the terminal.

Reviewer: ```suggestion
Ipyannotator's tests are directly integrated in the Jupyter Notebooks that make up the libraries source code, this is a consequence of following the literate programming style made possible by the library `nbdev`. Tests can be executed by running `nbdev_test_nbs` on the terminal.
```

> [Nbdev](https://nbdev.fast.ai/#A-Motivating-Example) it's the tool that allow Ipyannotator to be written in notebooks. The tool uses cell comments, such as `#exports`, to output the library code, and cells without comments to be executed as tests. Nbdev by itselft doesn't guarantee that the tests are self-contained, that's why Ipyannotator uses [pytest](https://docs.pytest.org/en/7.1.x/) and [ipytest](https://github.com/chmp/ipytest) as dev dependencies. To execute the tests the dev dependencies mentioned need to be installed.

Reviewer: move this paragraph to the bottom of this section, this background information is not need to run the tests

> When installing the repository using poetry, all dev dependencies are installed.

Reviewer: ```suggestion
When installing the repository using poetry, all dev dependencies are installed by default.
```

> When using pip make sure to install `pytest` and `ipytest`:

Reviewer: ```suggestion
When using pip for installation make sure to install the two dev dependencies `pytest` and `ipytest`, with the versions listed in `pyproject.toml`, manually:
```
